src/main/java/frc/robot/commands/DriveAndAim.java

```diff
@@ -35,11 +36,16 @@ public void initialize() {
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
-      shooter.aim(); //Set the angle of the shooter based on distance from target
+      double leftstickX = (Math.abs(xbox.getX(GenericHID.Hand.kLeft)) > Constants.xboxControllerDeadband) ? xbox.getX(GenericHID.Hand.kLeft): 0;
```

Comment:
I would recommend moving deadband calculations out of the command and into the controller or some utility class so every command can benefit from them. You don't want to have to copy-paste this I'm every command

Reply:
Oh, yeah, thanks for reminding me

Reply:
Check out my latest commit